lib/web_ui/lib/src/engine/view_embedder/application_dom/full_page_application_dom.dart

```diff
@@ -0,0 +1,128 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:js/js.dart';
+
+import '../../dom.dart';
+import '../../util.dart' show assertionsEnabled, setElementStyle;
+import 'application_dom.dart';
+
+
```

Comment:
Lots of DartDoc missing, why isn't analyzer shouting at me?

Reply:
`/engine/**/*.dart` might be exempt? Or maybe the whole web engine is 🤔